src/content/profile-tree.js

```diff
@@ -109,16 +120,21 @@ class ProfileTree {
     if (node === undefined) {
       const funcIndex = this._funcStackTable.func[funcStackIndex];
       const funcName = this._stringTable.getString(this._funcTable.name[funcIndex]);
+      const resourceIndex = this._funcTable.resource[funcIndex];
+      const resourceType = this._resourceTable.type[resourceIndex];
```

Comment:
I don't really like this, now that some of this moved to `_getOriginAnnotation` but I couldn't find a better idea.